src/Sarif.FunctionalTests/SpecExamples/Comprehensive.sarif

```diff
@@ -0,0 +1,107 @@
+{
+  "version": "1.0.0",
+  "$schema": "http://json.schemastore.org/sarif-1.0.0",
+  "runs": [
+    {
+      "id": "BC650830-A9FE-44CB-8818-AD6C387279A0",
+      "stableId": "Nightly code scan",
+      "baselineId": "0A106451-C9B1-4309-A7EE-06988B95F723",
+      "automationId": "Build-14.0.1.2-Release-20160716-13:22:18",
+      "architecture": "x86",
+      "tool": {
+        "name": "CodeScanner",
+        "fullName": "CodeScanner 1.1 for Unix (en-US)",
+        "version": "2.1",
+        "semanticVersion": "2.1.0",
+        "fileVersion": "2.1.0.0",
+        "language": "en-US",
+        "sarifLoggerVersion": "1.25.0",
+        "properties": {
+          "copyright": "Copyright (c) 2016 by Example Corporation. All rights reserved."
+        }
+      },
+      "invocation": {
+        "commandLine": "CodeScanner @collections.rsp",
+        "responseFiles": {
+          "collections.rsp": "-input src/collections/*.cpp -log out/collections.sarif -rules all -disable C9999"
+        },
+        "startTime": "2016-07-16T14:18:25Z",
+        "endTime": "2016-07-16T14:19:01Z",
+        "machine": "BLD01",
+        "account": "builder",
+        "processId": 1218,
+        "fileName": "/bin/tools/CodeScanner",
+        "workingDirectory": "/user/builder/work/src",
+        "environmentVariables": {
+          "PATH": "/usr/local/bin:/bin:/bin/tools:/home/builder/bin",
+          "HOME": "/user/builder",
+          "TZ": "EST"
+        }
+      },
+      "files": {
+        "file:///user/builder/work/src/collections/list.cpp": {
+          "mimeType": "text/x-c"
+        }
+      },
+      "logicalLocations": {
+        "collections::list::add": {
+          "name": "add",
+          "kind": "function",
+          "parentKey": "collections::list"
+        },
+        "collections::list": {
+          "name": "list",
+          "kind": "type",
+          "parentKey": "collections"
+        },
+        "collections": {
+          "name": "collections",
+          "kind": "namespace"
+        }
+      },
+      "results": [
+        {
+          "ruleId": "C2001",
+          "message": "Variable \"n\" was used without being initialized.",
```

Comment:
This almost looks like a line-break, \n due to escaping. :) suggest altering name slightly. if you use single quote, then the SARIF is more readable. for most internal tools, we use single quotes.